forch/forch_metrics.py

```diff
@@ -36,26 +36,45 @@ def stop(self):
         """Kill varz server"""
         self._http_server.stop_server()
 
-    def update_var(self, var, value):
-        """Update given varz with new value"""
+    def _get_varz(self, var):
         varz = self._metrics.get(var)
         if not varz:
             LOGGER.error('Error updating to varz %s since it is not known.', var)
             raise RuntimeError('Unknown varz')
+        return varz
+
+    def update_var(self, var, value):
+        """Update given varz with new value"""
+        varz = self._get_varz(var)
         if isinstance(varz, Info):
             varz.info(value)
         else:
             LOGGER.error('Error updating to varz %s since it\'s type %s is not known.',
                          var, type(varz))
             raise RuntimeError('Unknown varz type')
 
+    def inc_var(self, var, value=1):
+        """Increment Counter or Gauge variables"""
+        varz = self._get_varz(var)
+        # TODO: If a Gauge var is ever added, update to check for Gauge type
+        if isinstance(varz, Counter):
+            varz.inc(value)
+        else:
+            LOGGER.error('Error incrementing varz %s since it\'s type %s is not known.',
```

Comment:
nit -- you can include the error message in the raise and it pretty much has the same result